translations/cs/LC_MESSAGES/wesnoth-ala.po

```diff
@@ -263,27 +293,36 @@ msgid ""
 "the highest grade, and when I got here, less than a year ago, the commander "
 "of the... local garrison was... the private Titus."
 msgstr ""
+"Obávám se, že tady v Amoranu se už dlouho žádný důstojník nevyskytl. Jako "
+"seržant jsem zde nejvyšší hodnost. Když jsem se sem před necelým rokem "
+"dostal, velitelem... místní posádky byl... vojín Titus."
 
 #. [message]: speaker=Geobard
 #: A_Little_Adventure/utils/ALA_dialogues.cfg:125
 msgid ""
 "Incredible! Anyway, I don't think I'm wrong by putting me on your orders, "
 "commander...?"
 msgstr ""
+"Neuvěřitelné! Každopádně si myslím, že se nemýlím, že teď spadám pod vaše "
+"rozkazy, veliteli...?"
 
 #. [message]: speaker=Glauco
 #: A_Little_Adventure/utils/ALA_dialogues.cfg:134
 msgid ""
 "Sergeant Glauco at your service, sir cavalier, and I hope to be worthy of "
 "your confidence! And now let's rid out this rabble!"
 msgstr ""
+"Seržant Glauko k vašim službám, pane! Doufám, že budu hoden vaší důvěry! A "
```

Comment:
Mě to znělo trochu divně (pane jezdče), takže jsem to explicitní vyjádření vynechal a nechal jen pane.

Reply:
Kdyby tam byla hodnost, tak nic proti tomu, neboť to se běžně používá. Ale že by se ve vojenských kruzích používal třeba obrat pane tankisto, pane letče... ... no nevím.

Reply:
These are unit names, therefore the explicity.

Glauco looks up to them, so it doesn't sound weird to me.

You may leave the addressing without unit name, that was only my suggestion.

Reply:
So, I decided to omit the unit designation because it still sounds better to me.
Anyway, according to the scenario settings, Geobard will never appear as the cavalier, only as the dragoon (on easy) or the cavalryman (on normal and difficult). Maybe, it is a mistake?